tests/metrics/gha-run.sh

```diff
@@ -91,12 +91,32 @@ function run_test_launchtimes() {
 	bash tests/metrics/time/launch_times.sh -i public.ecr.aws/ubuntu/ubuntu:latest -n 20
 }
 
+function run_test_memory_usage() {
+	hypervisor="${1}"
+	info "Running memory-usage test using ${hypervisor} hypervisor"
+
+	create_symbolic_links "${hypervisor}"
+	bash tests/metrics/density/memory_usage.sh 20 5
+}
+
+function run_test_memory_usage_inside_container() {
+	hypervisor="${1}"
+	info "Running memory-usage inside the container test using ${hypervisor} hypervisor"
+
+	create_symbolic_links "${hypervisor}"
+	bash tests/metrics/density/memory_usage_inside_container.sh 5
+}
+
 function main() {
 	action="${1:-}"
 	case "${action}" in
 		install-kata) install_kata ;;
 		run-test-launchtimes-qemu) run_test_launchtimes "qemu" ;;
 		run-test-launchtimes-clh) run_test_launchtimes "clh" ;;
+		run-test-memory-usage-qemu) run_test_memory_usage "qemu" ;;
+		run-test-memory-usage-clh) run_test_memory_usage "clh" ;;
+		run-test-memory-usage-inside-container-qemu) run_test_memory_usage_inside_container "qemu" ;;
+		run-test-memory-usage-inside-container-clh) run_test_memory_usage_inside_container "clh" ;;
```

Comment:
Ideally, we'd short-circuit this to return success and not execute the tests.

This would ensure that tests would be green when added (of course, they'd do nothing), and then you could enable them by changing the script (which is taken into consideration by GHA, differently than changes in the yaml file themselves).

I'd prefer having this done in the suggested way,  but I'll leave it up to you to decide whether you want to follow the suggestion or not.

Reply:
@fidencio I took your suggestion and all the tests exits w/ zero.